.all-contributorsrc

```diff
@@ -0,0 +1,25 @@
+{
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": false,
+  "contributors": [
+    {
+      "login": "eleith",
+      "name": "eleith",
+      "avatar_url": "https://avatars.githubusercontent.com/u/284832?v=4",
+      "profile": "http://eleith.com",
+      "contributions": [
+        "infra"
```

Comment:
```suggestion
        "infra",
        "docs"
```

Reply:
Seems like this change isn't picked up yet

![image](https://user-images.githubusercontent.com/20406381/124604616-64dcf000-de6b-11eb-8ff8-5a522a089ff1.png)

Looking at the documentation this will be added when the next all-contributors generate run takes place (probably when the next interaction/add with the bot is triggered)